src/main/clojure/jarl/builtins/time.clj

```diff
@@ -1,23 +1,102 @@
 (ns jarl.builtins.time
   (:require [jarl.builtins.utils :refer [check-args]]
             [jarl.exceptions :as errors]
-            [jarl.utils :refer [instant-to-ns]])
-  (:import (java.time Instant ZoneId LocalDateTime ZoneOffset ZonedDateTime)
+            [jarl.utils :refer [instant-to-ns ns-to-instant]]
+            [clojure.string :as str])
+  (:import (java.time ZoneId LocalDateTime ZoneOffset ZonedDateTime Duration)
            (java.time.format TextStyle DateTimeFormatter)
-           (java.util Locale)))
+           (java.util Locale)
+           (java.time.temporal ChronoUnit)))
 
-(defn parse-iso-datetime [s]
-  (-> s
-      (LocalDateTime/parse DateTimeFormatter/ISO_LOCAL_DATE_TIME)
-      (.toInstant ZoneOffset/UTC)))
+; Courtesy of https://github.com/newm4n/go-dfe
+(def go->java-map
+  {; Go          Java
+   "January"   "MMMM"
+   "Jan"       "MMM"
+   "1"         "M"
+   "01"        "MM"
+   "Monday"    "EEEE"
+   "Mon"       "EEE"
+   "2"         "d"
+   "_2"        "_d"
+   "02"        "dd"
+   "15"        "HH"
```

Comment:
The golang format is bonkers .. 
Of course it's easier to remember `15` as opposed to `HH`, if you're a numberphile ..

Reply:
Not to mention reading the format.
Is this the American format: `01/02 '06`;
or is this: `02/01 '06`.
If you guess they guy who wrote the standard is an American, you might have a better chance than 50/50 😄 .

Reply:
Yeah, that format is absolutely insane IMHO.